comicScraper.py

```diff
@@ -109,6 +104,10 @@ def scrapeImageLinksFromIssue(url):
             imageUrl = extractImageUrlFromText(line)
             imageLinks.append(imageUrl)
 
+        if checkForCaptcha(line,url):
```

Comment:
does this get called for every image link?

Reply:
It gets called for every line in the soup from the issue link